pkg/features/kube_features.go

```diff
@@ -163,6 +163,7 @@ const (
 
 	// owner: @alculquicondor
 	// beta: v1.20
```

Comment:
Alpha is also forgotten.
+ alpha: v1.19

Reply:
Good catch. Thanks Alex.